_yaml/data-functions.yml

```diff
@@ -212,6 +230,15 @@ df_number:
         description: How many decimal places to round to.
     returns: Number
     description: Returns the value of a number rounded to the nearest whole number, unless a decimal place is specified.
+  - funcName: toBoolean
+    returns: Boolean
+    description: Converts a number to a boolean. <code>0</code> converts to <code>false</code>. All other values convert to <code>true</code>.
+  - funcName: toDateTime
+    returns: Date
+    description: Converts a number to a <a href="https://docs.n8n.io/code/cookbook/luxon/">Luxon date object</a>.
```

Comment:
Could use some detail on the argument, from autocomplete description:

"Converts a number to a DateTime. Defaults to milliseconds. Format can be 'ms' (milliseconds), 's' (seconds) or 'excel' (Excel 1900 format)."